backend/src/main/java/com/woowacourse/levellog/team/exception/DuplicateWatchersException.java

```diff
@@ -0,0 +1,13 @@
+package com.woowacourse.levellog.team.exception;
+
+import com.woowacourse.levellog.common.exception.LevellogException;
+import org.springframework.http.HttpStatus;
+
+public class DuplicateWatchersException extends LevellogException {
```

Comment:
```suggestion
public class WatchersDuplicateException extends LevellogException {
```
`XXXNotFoundException` 갬성으로 가면 어떨까요?

Reply:
해당 예외를 `InvalidFieldException`으로 던지면서 현재는 삭제되었습니다~!